README.md

```diff
@@ -77,7 +77,8 @@ With all of the prerequisites installed, run the following commands to
 configure the `nvidia-container-toolkit` for use with `kind`.
 
 ```bash
-sudo nvidia-ctk runtime configure --runtime=docker --set-as-default --cdi.enabled
+sudo nvidia-ctk runtime configure \
+    --runtime=docker --set-as-default --cdi.enabled --config-source=command
```

Comment:
This should be:
```suggestion
sudo nvidia-ctk runtime configure \
    --runtime=docker --set-as-default --cdi.enabled
```
For `docker` -- which is what we're configuring to get kind running with GPUs at all -- the `--config-source` should still be `file` (the default). Note that `nvidia-ctk runtime configure` handles this for docker specifically: https://github.com/NVIDIA/nvidia-container-toolkit/blob/92472bd0edc1ab9b96bf2b48827cb96c0f7b38f8/cmd/nvidia-ctk/runtime/configure/configure.go#L216-L226

Reply:
done